homeassistant/components/switchbot/coordinator.py

```diff
@@ -0,0 +1,59 @@
+"""Provides the switchbot DataUpdateCoordinator."""
+from __future__ import annotations
+
+from asyncio import Lock
+from datetime import timedelta
+import logging
+
+import switchbot  # pylint: disable=import-error
```

Comment:
```suggestion
import switchbot
```

This should't be needed

Reply:
The library is "force commented" from the requirements file. According to @frenck this is done due to issues with hassio commit tests workflow.

Reply:
Do you know what the issue is? Can we fix it in the upstream library to avoid this problem?

Reply:
I think it's related to bluepy. I see all of the integrations making use of bluepy are commented out.

# homeassistant.components.decora
# homeassistant.components.miflora
# bluepy==1.3.0

The PySwitchbot library makes use of bluepy.